admin_pages/registrations/templates/reg_admin_register_new_attendee_step_content.template.php

```diff
@@ -23,6 +23,10 @@
                 <label for="txn_reg_status_change" class="last">
                     <?php esc_html_e('Send Related Messages?', 'event_espresso'); ?>
                 </label>
+                <input name="txn_reg_status_change[send_notifications]"
+                       type="hidden"
+                       value="0"
+                />
```

Comment:
this is a great approach and i wish we had used this from the start, but adding this in now could easily cause issues.

How, you say?

Well if there is any code that assumes that the mere presence of the `send_notifications` is enough to trigger that logic, without checking the value, then this would throw off that logic. 

In other words, currently we either:
- HAVE a `send_notifications` request param, meaning "trigger notices"
 or
- NO `send_notifications` request param, meaning "do NOT trigger notices"

It might look ok to do that within the core plugin (and it most certainly is) but we also have a ton of other addons that would need to be checked, AND you also have to remember that EE is a distributed product so there could potentially be third party or user customizations out there that are relying on the above logic and this change could break that.